tests/map.cpp

```diff
@@ -177,7 +179,7 @@ TEST_CASE("Simple map serialization/deserialization")
     origin.ints.append("key1", 12);
     origin.ints.append("key2", 13);
 
-    auto check = [](const TestMap& item) {
+    auto check = [](TestMap& item) {
```

Comment:
```suggestion
    auto check = [](const TestMap& item) {
```

Reply:
Sorry, it has been yet taken into account, the feature has not been updated